e2e/app.e2e-spec.ts

```diff
@@ -1,14 +1,14 @@
-import { RasterFoundryDocsPage } from './app.po';
+import { RasterFoundryApiDocsPage } from './app.po';
 
-describe('raster-foundry-docs App', () => {
-  let page: RasterFoundryDocsPage;
+describe('raster-foundry-api-docs App', () => {
+  let page: RasterFoundryApiDocsPage;
 
   beforeEach(() => {
-    page = new RasterFoundryDocsPage();
+    page = new RasterFoundryApiDocsPage();
   });
 
   it('should display message saying app works', () => {
     page.navigateTo();
-    expect(page.getParagraphText()).toEqual('Main app component');
+    expect(page.getParagraphText()).toEqual('app works!');
```

Comment:
Let's put some real tests in. We could probably have this actually test with a cut down swagger spec to make sure features work.